packages/wp-source/src/actions.ts

```diff
@@ -122,6 +122,14 @@ const actions: WpSource["actions"]["source"] = {
 
     libraries.source.api.init({ api, isWpCom });
 
+    // If the URL contains an auth token, then add it to the state.
+    // This is normally the case e.g, when accessing the post preview.
+    const auth = state.frontity?.options?.frontity_source_auth;
+    const authFromEnv = process.env.FRONTITY_SOURCE_AUTH;
```

Comment:
Just to be sure, have you checked that the token doesn't end up in the `server.js` bundle?

It shouldn't, but I'd prefer to check it out to be sure 😅

Reply:
nope, it doesn't end up in the bundle.

I've set the env variable locally to something like `1234567` and just manually searched the bundle for it.

Reply:
Great, thanks.